### DISCUSSION TITLE
Microsoft office files detected as zip

### DISCUSSION THREAD

**Discussion by Shimon456** [horsicq/Detect-It-Easy#145] [Q&A]
Hi,
When I run DIE tool with Microsoft Office files (.docx, .pptx, etc.) it detects them as zip files.
Is there any way to classify Microsoft Office files better?

An example of .docx file with only text in it:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/33197548/214428043-3655bdcf-d82a-421e-92bb-f162364b6a95.png">

Thanks.

**Comment by horsicq** [comment]
Hello! Thanks a lot for the bugreport! I will fix it.

**Comment by BJNFNE** [comment]
Hello, I will look into it

**Comment by BJNFNE** [comment]
@Shimon456 I looked in the Issue. It seems that Office files are basically ZIP files. You can exactly extract and docx/pptx with unzip. :)
